pypit/artrace.py

```diff
@@ -656,11 +610,11 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False):
             lcnt = 1
         else:
             msgs.error("Unable to trace any edges"+msgs.newline()+"try a different method to trace the order edges")
-    elif (rcnt == 0) and (lcnt == 1):
+    elif rcnt == 0:
```

Comment:
Perhaps it was needed by the old tracing algorithm? This implementation works with the test suite. In fact, there was a case where rcnt=0 and lcnt=2, (one of the lcnt was spurious, and the right edge was butted against the edge of the detector in LRIS).